homebrew_challenge/src/components/Facts/Facts.css

```diff
@@ -0,0 +1,71 @@
+.icons-row {
```

Comment:
Are the facts files supposed to be in this PR ?

Reply:
No will undo

Reply:
resolved